functionaltests/src/test/resources/specs/scheduler/ScheduledJobShort.html

```diff
@@ -76,7 +76,9 @@ <h4>Wait <span c:echo="waitSecondsTimeout()"></span> seconds and verify the paym
             <th c:assertEquals="#verifyResult.paymentStatus">Payment Status</th>
             <th c:assertEquals="#verifyResult.interfaceInteractionsSize">Interface Interactions</th>
             <th c:assertEquals="#verifyResult.transactionsSize">Transactions</th>
-            <th c:echo="#verifyResult.lastModifiedAt">Interface Interactions</th>
+            <th c:echo="#verifyResult.createdAt">Created At (FIO)</th>
```

Comment:
nitpick: Usually (words with less than 4 letters e.g. most prepositions) should not be capitalised in titles ;) https://english.stackexchange.com/questions/14/which-words-in-a-title-should-be-capitalized

this is not imp. tho.

Reply:
a284d4d4